lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -72,6 +73,10 @@ def each_source
 
             source_type = source_types_by_id[source["source_type_id"]]
 
+            if AVAILABILITY_CHECK_SOURCE_TYPES.include?(source_type["name"])
```

Comment:
can be written as `source_type.has_availability_check?` 
and moved there: https://github.com/ManageIQ/topological_inventory-orchestrator/blob/master/lib/topological_inventory/orchestrator/worker.rb#L116